### PR TITLE
Fix csv importing

### PR DESCRIPTION
**Pull request by dmfenton**
Fixes csv import logic that was broken in the refactor

### CODE REVIEW

**Comment by dmfenton** [comment]
ping @ngoldman this is ready for review

File: index.js [hunk 1]
```diff
@@ -2,7 +2,6 @@ var pjson = require('./package.json')
 
 exports.name = 'agol'
 exports.hosts = true
-exports.pattern = '/:id/:item'
```
Comment (ngoldman) [inline index.js]: @dmfenton no more pattern?

File: routes/index.js [hunk 2]
```diff
@@ -18,5 +18,9 @@ module.exports = {
   'get /agol/:id/:item/thumbnail/:layer': 'thumbnail',
   'get /agol/:id/:item/:layer/tiles/:z/:x/:y.:format': 'tiles',
   'get /agol/:id/:item/:layer/tiles/:z/:x/:y': 'tiles',
+  'get /agol/:id/:item/FeatureServer/:layer': 'featureserver',
+  'get /agol/:ite/:item/FeatureServer/:layer/:method': 'featureserver',
+  'post /agol/:ite/:item/FeatureServer/:layer/:method': 'featureserver',
```
Comment (ngoldman) [inline routes/index.js]: @dmfenton what is `:ite`

File: index.js (hunk 1 above)
Comment (dmfenton) [inline index.js]: correct.
Two reasons:
1. pattern is not well understood or documented
2. it prevents me from using the shared controller method to pass through information

File: routes/index.js (hunk 2 above)
Comment (dmfenton) [inline routes/index.js]: a typo :crying_cat_face:

File: index.js (hunk 1 above)
Comment (ngoldman) [inline index.js]: 1. true
2. alright

we should probably either flesh out the pattern idea and document it or remove it from providers

Comment (dmfenton) [inline index.js]: agreed.

**Comment by ngoldman** [comment]
:shipit:

**Pull request merged**